runtime/interface.go

```diff
@@ -102,138 +179,131 @@ type Interface interface {
 	) (bool, error)
 	// Hash returns the digest of hashing the given data with using the given hash algorithm
 	Hash(data []byte, hashAlgorithm string) ([]byte, error)
-	// GetStorageUsed gets storage used in bytes by the address at the moment of the function call.
-	GetStorageUsed(address Address) (value uint64, err error)
-	// GetStorageCapacity gets storage capacity in bytes on the address.
-	GetStorageCapacity(address Address) (value uint64, err error)
-}
-
-type HighLevelStorage interface {
-	Interface
-
-	// HighLevelStorageEnabled should return true
-	// if the functions of HighLevelStorage should be called,
-	// e.g. SetCadenceValue
-	HighLevelStorageEnabled() bool
-
-	// SetCadenceValue sets a value for the given key in the storage, owned by the given account.
-	SetCadenceValue(owner Address, key string, value cadence.Value) (err error)
 }
 
 type Metrics interface {
+	// ProgramParsed captures the time spent on parsing the program
 	ProgramParsed(location common.Location, duration time.Duration)
+	// ProgramChecked captures the time spent on checking the parsed program
 	ProgramChecked(location common.Location, duration time.Duration)
+	// ProgramInterpreted captures the time spent on interpreting the parsed and checked program
 	ProgramInterpreted(location common.Location, duration time.Duration)
+
+	// ValueEncoded captures the time spent on encoding a value
+	// TODO: maybe add type
 	ValueEncoded(duration time.Duration)
+	// ValueDecoded capture the time spent on decoding an encoded value
+	// TODO: maybe add type
 	ValueDecoded(duration time.Duration)
 }
 
-type EmptyRuntimeInterface struct{}
-
-var _ Interface = &EmptyRuntimeInterface{}
+type HighLevelAccountStorage interface {
+	// HighLevelStorageEnabled should return true
+	// if the functions of HighLevelStorage should be called,
+	// e.g. SetCadenceValue
+	HighLevelStorageEnabled() bool
 
-func (i *EmptyRuntimeInterface) ResolveLocation(identifiers []Identifier, location Location) ([]ResolvedLocation, error) {
-	return []ResolvedLocation{
-		{
-			Location:    location,
-			Identifiers: identifiers,
-		},
-	}, nil
+	// SetCadenceValue sets a value for the given key in the storage, owned by the given account.
+	SetCadenceValue(owner Address, key string, value cadence.Value) (err error)
```

Comment:
Maybe keep consistency with the "low-level" storage interface: Maybe replace the `owner` and `key` parameters with a storage key parameter, like you did for the other storage functions above.